src/components/core/icons.js

```diff
@@ -208,18 +208,39 @@ export const WarningIcon = (props) => (
   <svg
     {...props}
     xmlns="http://www.w3.org/2000/svg"
-    width="24"
-    height="24"
-    viewBox="0 0 24 24"
+    width="22"
+    height="19"
+    fill="none"
   >
-    <path d="M0 0h24v24H0z" fill="none" />
     <path
-      d="M1 21h22L12 2 1 21zm12-3h-2v-2h2v2zm0-4h-2v-4h2v4z"
-      fill="currentColor"
+      fill="#C1002B"
+      d="M1.73 19h18.53c.77 0 1.25-.83.87-1.5l-9.27-16a.996.996 0 0 0-1.73 0l-9.27 16c-.38.67.1 1.5.87 1.5ZM12 16h-2v-2h2v2Zm-1-4c-.55 0-1-.45-1-1V9c0-.55.45-1 1-1s1 .45 1 1v2c0 .55-.45 1-1 1Z"
     />
   </svg>
 );
 
+export const ArrowDownIcon = (props) => (
+  <svg
+    {...props}
```

Comment:
If it has to be that way, I'd pass `{...props}` as the last prop. Imagine if someone wanted to overwrite the height, then it would be impossible because `height="20"` comes after `{...props}`.